tests/automation_framework/src/work_order_submit/work_order_submit_params.py

```diff
@@ -38,6 +38,7 @@ def __init__(self):
         self.request_mode = "file"
         self.tamper = {"params": {}}
         self.output_json_file_name = "worker_submit"
+        self.final_hash = ""
```

Comment:
where are we using this variable?

Reply:
test_workordersubmit_requesterSignature_noencryptedRequestHash test, removed EncryptedRequestHash parameter from input json and got below error. 
src/work_order_submit/work_order_submit_params.py:548: in compute_signature
    self._compute_requester_signature()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <src.work_order_submit.work_order_submit_params.WorkOrderSubmit object at 0x7f33e0aa8cf8>

    def _compute_requester_signature(self):
        self.public_key = self.private_key.GetPublicKey().Serialize()
>       signature_result = self.private_key.SignMessage(self.final_hash)
E       AttributeError: 'WorkOrderSubmit' object has no attribute 'final_hash
So corrected the framework to run this test. Its not impacting other tests as its unique scenario, not checked before. Also ran all tests to verify and its not altering other tests.